source/tests/system/nifgen_session_tests.cpp

```diff
@@ -21,15 +21,10 @@ const char* kTestInvalidFgenRsrc = "";
 class NiFgenSessionTest : public ::testing::Test {
  protected:
   NiFgenSessionTest()
+    : device_server_(DeviceServerInterface::Singleton()),
```

Comment:
@doshirohan , we need to update fgen driver api tests also know?

Reply:
Somehow I couldn't find then in VS, I thought they didn't exist. I will update them in a bit.

Reply:
Updated api tests as well.